cfme/tests/cloud_infra_common/test_genealogy.py

```diff
@@ -68,7 +68,7 @@ def test_vm_genealogy_detected(
             vm_crud_ancestors = create_vm.genealogy.ancestors
         except NameError:
             logger.exception("The parent template not detected!")
-            raise pytest.fail("The parent template not detected!")
+            pytest.fail("The parent template not detected!")
```

Comment:
As discussed, this whole try/except block may get removed with other test automation work, because there does not appear to be any path for a NameError during the call to `genealogy.ancestors`.

Good to clean this up to a direct `pytest.fail` call for now.